packages/web3/src/web3.ts

```diff
@@ -31,7 +31,7 @@ import { initAccountsForContext } from './accounts';
 import { Web3EthInterface } from './types';
 import { Web3PkgInfo } from './version';
 
-export class Web3 extends Web3Context<EthExecutionAPI> {
+export default class Web3 extends Web3Context<EthExecutionAPI> {
```

Comment:
Just out of curiosity, is this part of the issue fixing, or is it an enhancement?

Reply:
Its required for this issue fixing.